cassandra/src/main/java/se/tre/freki/storage/cassandra/CassandraStore.java

```diff
@@ -415,14 +415,14 @@ public Boolean apply(final ResultSet update) {
 
         final Row row = result.get();
 
-        if (Strings.isNullOrEmpty(row.getString("meta_description"))) {
+        if (Strings.isNullOrEmpty(row.getString("description"))) {
           return Optional.absent();
         }
 
         return Optional.of(
             LabelMeta.create(CassandraLabelId.fromLong(row.getLong("label_id")),
-                LabelType.fromValue(row.getString("type")), row.getString("name"),
-                row.getString("meta_description"),
+                LabelType.fromValue(row.getString("type")), "REMOVE THIS",
```

Comment:
Can you change this back to read the name until the column is actually removed? This will not compile or run once that starts to be removed.